### ISSUE TITLE
Can I fetch from 2 API from different URLs ?

### ISSUE THREAD

**Issue by buigiaanfb1** [reduxjs/redux-toolkit#1314]
My problem is can I use createApi with baseUrl in URLs ? I mean i will create 2 files with 2 setup with different URLs but it not work the first API i use for authentication it works perfectly but the second return undefinded but i see it on redux store not on the reducer that i created in slice.
Thank you for reading my English is not good!!
<img width="320" alt="Screen Shot 2021-07-17 at 04 01 22" src="https://user-images.githubusercontent.com/36767199/126008156-7f90ac54-1f7e-4b54-85e7-5ab5f70e2d02.png">

**Comment by phryneas** [comment]
You can have multiple apis, but have to hook all of them up with their reducer and middleware - and they have to have different `reducerPath` options in your `createApi` call.

Usually though, you will want to have *one* `createApi` call with multiple endpoints for your whole application unless you are really having completely unrelated services, so really fetching data from different domains.

**Comment by buigiaanfb1** [comment]
> You can have multiple apis, but have to hook all of them up with their reducer and middleware - and they have to have different `reducerPath` options in your `createApi` call.
> 
> Usually though, you will want to have _one_ `createApi` call with multiple endpoints for your whole application unless you are really having completely unrelated services, so really fetching data from different domains.

yeah. because my project in in staging so i need my own Backend Url and everyone is like me, as i read your comment, maybe i made a mistake is dont't have middleware for that, let me try again, 
Thank you for your fast reply!!!! Have a nice day
